docs/sources/contribute/_index.md

```diff
@@ -21,26 +21,23 @@ weight: 200
 There are a number of different ways to contribute to documentation at Grafana Labs.
 Choose the one that applies best and learn how you can engage with the documentation team and help by contributing your ideas.
 
-## Report a problem
+## {{% translate.inline "docs_feedback_report" %}}{{ markdownify (T (.Get 0)) }}{{% /translate.inline %}}
```

Comment:
Any reason we cannot just use `{{% translate.inline "docs_feedback_report" /%}}` here and below? I am not sure we want to promote the use of `markdownify` in content. 

Also, TIL about the translate shortcode, should we add to /docs/writers-toolkit/contribute/?

Reply:
Good point about the use of `markdownify` and I'm glad you asked about it because I'm sure that we don't want to be encouraging it now that I've thought about it.

The reason was I spotted the Markdown in one of the translations https://github.com/grafana/website/pull/22302/files#diff-360ec96d474fa3fc5b356989607caae93e55b6183677d569dc68ea547427d8cdR57 which is then rendered in https://github.com/grafana/website/pull/22302/files#diff-b10673a8041efa210d39d2de3910e39470d74915ee573ed2fd1da4ca5a06afccR23.

I think we probably want to keep the translation strings free of Markdown too?

Reply:
I've also actually cheated here by using an [inline shortcode](https://gohugo.io/templates/shortcode/#inline-shortcodes).

If we like the idea of it, I should really make it a proper shortcode and document it as you've proposed

Reply:
Ah yes, fixed in https://github.com/grafana/website/pull/22302/commits/842226cebc8f9e0c3648c759ce09da35900aff8b.